ibc-core/ics23-commitment/types/src/commitment.rs

```diff
@@ -127,10 +122,16 @@ impl TryFrom<CommitmentProofBytes> for MerkleProof {
     type Error = CommitmentError;
 
     fn try_from(value: CommitmentProofBytes) -> Result<Self, Self::Error> {
```

Comment:
I wonder if we should keep this conversion from owned as well? looks redundant!

Reply:
I’ve no opinion about converting to and from MerkleProof.

For conversion to and from Vec my intuition tells me we do want to keep them since that helps avoiding allocations.